src/Modules/All/Commands.All/Commands.All.csproj

```diff
@@ -29,7 +29,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\**\*.csproj" Exclude="..\..\**\$(MSBuildThisFile);..\..\**\*.Test.csproj" />
+    <ProjectReference Include="..\..\Data\Commands.Data\Commands.Data.csproj" />
```

Comment:
If it were my change, I would keep everything building and comment out the CmdletsToExport as done below. In any case, flipping this back is just a git revert away, so it doesn't make a huge difference.